css/style.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+.body {
+    margin: 0;
+    box-sizing: border-box;
+    scroll-behavior: smooth;
+}
+
+#navbar {
+    background-color: bisque;
+    padding: 0;
+    margin: 0;
+    font-family: Poppins;
+    position: fixed;
+    top: 0;
+    width: 100%;
+}
+
+ul {
+    display: flex;
+    list-style: none;
+    justify-content: end;
+    font-size: 18px;
+    padding-top: 10px;
+    padding-bottom: 10px
+}
+
+li {
+    padding-right: 2em;
+    cursor: pointer;
+}
+
+#welcome-section {
+    height: 100vh;
+    background-color: #2A1915;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+h1 {
+    font-size: 65px;
+    font-family: Poppins;
+}
+
+@media(max-width: 576px) {
+    h1 {
+        font-size: 35px;
+    }
+}
+
+p {
+    font-size: 23px;
+    font-family: Poppins;
```

Comment:
I think it would be better if you add two or three fallback fonts. There is always a chance that a font is not found or is not installed properly in some browsers. Therefore, it is very important to always use fallback fonts.